play-media-native/screens/MediaDetail.tsx

```diff
@@ -0,0 +1,124 @@
+import { faEdit } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { Image, ScrollView, StyleSheet, View } from 'react-native';
+import { AnimatedFAB, Text } from 'react-native-paper';
+import { useQuery } from 'react-query';
+
+import { getMediaById } from '../api/queries/getMedia';
+import { LoadingScreen } from '../features/LoadingScreen/LoadingScreen';
+import { Screen } from '../features/Screen/Screen';
+import { useScrollOffset } from '../hooks/useScrollOffset/useScrollOffset';
+import { styles } from '../theme/styles';
+import { theme } from '../theme/theme';
+
+const pageStyles = StyleSheet.create({
+  title: {
+    fontFamily: theme.fontFamily.bold,
+    marginBottom: theme.spacing.xxs,
+  },
+  body: {
+    marginBottom: theme.spacing.sm,
+  },
+  bottomFAB: {
+    position: 'absolute',
+    right: theme.spacing.sm,
+    bottom: theme.spacing.xs,
+  },
+  button: {
+    position: 'absolute',
+    right: -theme.spacing.sm,
+    top: -theme.spacing.xs,
+  },
+  actionBtns: {
+    paddingBottom: 0,
+    paddingRight: theme.spacing.xs,
+  },
+});
+
+export const MediaDetailScreen = ({ route, navigation }) => {
+  const id = route?.params?.id;
+  const [error, setError] = useState<unknown>();
```

Comment:
Are you sure this state is needed? Could we use the error prop from useQuery itself?

const { data: media, error, isFetching } = useQuery